CMakeLists.txt

```diff
@@ -20,7 +20,13 @@ set(EXTENSION_DEPENDS SlicerIGSIO)
 # Extension dependencies
 find_package(Slicer REQUIRED)
 include(${Slicer_USE_FILE})
-find_package(SlicerIGSIO REQUIRED)
+
+if (NOT DEFINED Slicer_EXTENSION_SOURCE_DIRS)

```

Comment:
What happens if SlicerIGSIO is bundled but SlicerIGT is not? Does it still work?
Maybe it would be better to try `find_package(IGSIO)` and if it is not found then try finding `find_package(SlicerIGSIO REQUIRED)`?